guides/elements.md

```diff
@@ -377,14 +385,17 @@ You can pass options to the essence view.
 
 ~~~ erb
 <%= element_view_for(element) do |el| %>
-  <%= el.render :image, {image_size: '200x300'}, class: 'image-large' %>
+  <%= el.render :image, {size: '200x300'}, class: 'image-large' %>
 <% end %>
 ~~~
 
-::: tip The first hash is the `options` the second one the `html_options` passed to the wrapper of the content.
-If you only want to pass `html_options` you need to pass an empty hash as second argument.
+::: tip
+Instead of passing the `size` of an image into the `EssencePicture` as shown above, you should consider to use static [content settings](https://guides.alchemy-cms.com/essences.html#individual-essence-settings) instead.
```

Comment:
```suggestion
Instead of passing the `size` of an image into the `EssencePicture` as shown above, you should consider to use static [content settings](essences.html#individual-essence-settings) instead.
```